scylla-gdb.py

```diff
@@ -651,7 +651,7 @@ def __bool__(self):
 
 
 def unordered_map(ref):
-    if ref.type.name.startswith('flat_hash_map'):
+    if 'flat_hash_map' in ref.type.strip_typedefs().name:
```

Comment:
> Match the flat_hash_map at any position, not just the very start.

Why is this needed?

What if we have `unordered_map<sstring, flat_hash_map<int, int>>`, which is not a flat_hash_map ?